client/src/app/shared/components/list-of-speakers-content/list-of-speakers-content.component.html

```diff
@@ -316,12 +317,12 @@ <h1 class="los-title" *ngIf="!customTitle">
             </button>
 
             <!-- star button -->
-            <button mat-menu-item (click)="onMarkButton(speaker)">
+            <button mat-menu-item (click)="onMarkButton(speaker)" *ngIf="canSetMarkSelf">
```

Comment:
^ this. I do not think that this is what we want. As far as I understand the "Mark Self" feature is pretty much that: Marking myself. The LOS-manager should mark anything anytime.

Reply:
Ok, we get feedback on events thats admins are not sure if star is enabled or disabled for normal users. But I see it could be useful to set the star as admin only. I'll revert this change.